tests/epyccel/test_epyccel_sets.py

```diff
@@ -13,6 +13,7 @@
     ],
     scope = "module"
 )
+
```

Comment:
Useless change?

Reply:
Fixed : c20c7ed

Reply:
That is a different line

Reply:
Decorators shouldn't have a space between them and the function that is being passed to them as an argument

Reply:
Fixed : dc1b329